tests/scripts/thread-cert/Cert_7_1_08_BorderRouterAsFED.py

```diff
@@ -0,0 +1,210 @@
+        #         to the Leader.
+        pkts.filter_wpan_src64(FED).\
+            filter_ipv6_dst(LEADER_ALOC).\
+            filter_coap_request(SVR_DATA_URI).\
+            filter(lambda p: {
+                              Ipv6Addr(PREFIX_2001[:-3]),
+                              Ipv6Addr(PREFIX_2002[:-3])
+                             } == set(p.thread_nwd.tlv.prefix) and\
+                   p.thread_nwd.tlv.border_router_16 is not nullField
+                   ).\
+            must_next()
+
+        # Step 3: Leader transmits a 2.04 Changed CoAP response to the DUT.
+        #         And transmits multicast MLE Data Response with the new information
+        #         collected, adding also 6LoWPAN ID TLV for the prefix set on FED.
+        with pkts.save_index():
+            pkts.filter_wpan_src64(LEADER).\
+                filter_ipv6_dst(FED_RLOC).\
+                filter_coap_ack(SVR_DATA_URI).\
+                must_next()
+        with pkts.save_index():
+            pkts.filter_wpan_src64(LEADER).\
+                filter_LLANMA().\
+                filter_mle_cmd(MLE_DATA_RESPONSE).\
+                filter(lambda p: {
+                                  Ipv6Addr(PREFIX_2001[:-3]),
+                                  Ipv6Addr(PREFIX_2002[:-3])
+                                 } == set(p.thread_nwd.tlv.prefix) and\
+                       p.thread_nwd.tlv.border_router.flag.p == [1, 1] and\
+                       p.thread_nwd.tlv.border_router.flag.s == [1, 1] and\
+                       p.thread_nwd.tlv.border_router.flag.r == [1, 1] and\
+                       p.thread_nwd.tlv.border_router.flag.o == [1, 1] and\
+                       p.thread_nwd.tlv.stable == [0, 1, 1, 1, 0, 0, 0]
+                       ).\
+                must_next()
+
+        # Step 4: The DUT transmits multicast MLE Data Response with the new information
+        #         collected, adding also 6LoWPAN ID TLV for the prefix set on FED.
+        pkts.filter_wpan_src64(ROUTER).\
+            filter_LLANMA().\
+            filter_mle_cmd(MLE_DATA_RESPONSE).\
+            filter(lambda p: {
+                              Ipv6Addr(PREFIX_2001[:-3]),
+                              Ipv6Addr(PREFIX_2002[:-3])
+                             } == set(p.thread_nwd.tlv.prefix) and\
+                   p.thread_nwd.tlv.border_router.flag.p == [1, 1] and\
+                   p.thread_nwd.tlv.border_router.flag.s == [1, 1] and\
+                   p.thread_nwd.tlv.border_router.flag.r == [1, 1] and\
+                   p.thread_nwd.tlv.border_router.flag.o == [1, 1] and\
+                   p.thread_nwd.tlv.stable == [0, 1, 1, 1, 0, 0, 0]
+                   ).\
+            must_next()
+
+        # Step 6: The DUT notifies Leader of removed server’s (FED’s) RLOC16.
+        #         The DUT MUST send a CoAP Server Data Notification frame
+        #         to the Leader containing only the removed server’s RLOC16:
```

Comment:
need check that the TLVs `containing only the removed server’s RLOC16` without other TLVs?

Reply:
added the check